.jshintrc

```diff
@@ -13,7 +13,7 @@
     "globalstrict": true,
     "immed": true,
     "indent": 4,
-    "latedef": true,
+    "latedef": "nofunc",
```

Comment:
Yep, I don't really understand why this came up now, and didn't have time to investigate...